mysite/announcements/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+from .models import Announcements
+from django.contrib.auth.models import User
+
+class AnnouncementsAdmin(admin.ModelAdmin):
+    list_display = ('title', 'message', 'date_time_expire')
+    change_form_template = "announcements/announcements_change_form.html"
+
+    def response_change(self, request, obj):
+        if "_send_announcement" in request.POST:
+            users = User.objects.all()
+            for user in users:
+                print str(user) + ' ' + request.POST['title'] + ' ' +request.POST['message']
```

Comment:
So, we generally use [`.format`](https://pyformat.info/) for formatting strings vs using `%` or `+`. 
Simple reasons =>
- it looks more pretty.
- handles type conversion explicitly.